src/generators/static/readmeFileGenerator.ts

```diff
@@ -96,7 +96,14 @@ function createMetadata(
   const packageDirectoryName = names?.[1];
 
   const clientPackageName = packageDetails?.name;
-  const { clientClassName, serviceTitle } = getClientAndServiceName(codeModel.language, codeModel.info);
+  const { clientClassName, serviceTitle } = getClientAndServiceName(
+    codeModel.language,
+    codeModel.info
+  );
+  const simpleServiceNameTmp =
```

Comment:
Can't we just name it `serviceName`/`simpleServiceName`/`heading` or anything without Tmp?

Reply:
changed